.storybook/preview.js

```diff
@@ -27,3 +28,20 @@ for (let i = -3; i <= 6; i++) {
 }
 addDecorator(withPaddings);
 addParameters({ paddings });
+
+// Create viewports using widths defined in design tokens
+const breakpointViewports = Object.keys(breakpoints).map((name) => {
+  return {
+    name,
+    styles: {
+      width: breakpoints[name],
+      height: '100%',
+    },
+    type: 'other',
+  };
+});
+addParameters({
+  viewport: {
+    viewports: breakpointViewports,
```

Comment:
@Josh68 Could we prepend this to the existing list of minimal viewports? See: https://github.com/storybookjs/storybook/tree/master/addons/viewport#use-detailed-set-of-devices

Reply:
Yes, I wasn't sure whether we wanted that, but assumed we'd be replacing. I'll use the other approach